tests/config/astrometryReferenceErr-None-config.py

```diff
@@ -3,4 +3,5 @@
 # We have to use the sdss refcat, as gaia supplies coord errors.
```

Comment:
That is an interesting and rather disturbing pre-existing comment. Providing uncertainties is a problem?

Reply:
I've expanded the comment: the goal of this file is to test what happens when there are no coord errors in the refcat and no override is supplied (jointcal should fail with a useful error message).